helm/maistra-threescale/values.yaml

```diff
@@ -2,7 +2,7 @@ enabled: false
 
 hub: quay.io/3scale
 image: 3scale-istio-adapter
-tag: v0.4.1
+tag: v1.0.0
```

Comment:
Technically this should be updated in patch-charts.sh, but #176 is changing this.